src/routes/requestRoutes.js

```diff
@@ -5,19 +5,24 @@ import { validateTrips, validateAccommodation } from '../validation/trips';
 import reqMidd from '../middlewares/requestMiddlware';
 import userMidd from '../middlewares/userMiddlware';
 import roles from '../middlewares/rolesMiddlewares';
+import comment from '../controllers/comment';
 
 const app = express.Router();
 
 const { checkExistingTrip, checkLineManager, checkManagerId } = reqMidd;
 const { changeRequestStatus, getManagerRequests } = requestController;
-const { checkManager, checkRequester } = roles;
+const { checkManager, checkRequester, checkTripOwner } = roles;
 const { checkToken } = userMidd;
-const { tripValidation } = valid;
+const { tripValidation, addCommentValidation, viewCommentValidation } = valid;
+const { addComment, viewComment } = comment;
 
 app.get('/', checkToken, checkRequester, requestController.getRequest);
 /* eslint-disable max-len */
 app.post('/', checkToken, checkRequester, valid.request, validateTrips, validateAccommodation, requestController.postRequest);
 app.get('/managers/:managerId', checkToken, checkManager, checkManagerId, getManagerRequests);
 app.patch('/:requestId/:status', checkToken, checkManager, checkExistingTrip, checkLineManager, tripValidation, changeRequestStatus);
+app.put('/:requestId/comment', addCommentValidation, checkToken, checkExistingTrip, checkTripOwner, addComment);
```

Comment:
Let's have a separate route for the comments. Kindly use POST to create comments.